.packit.yaml

```diff
@@ -0,0 +1,15 @@
+downstream_package_name: anaconda-user-help
```

Comment:
This should be `downstream_package_name: blivet-gui`, right? And same with the `synced_files` below.

Reply:
Oh, there was a mixup. I just mindlessly got the source from here https://src.fedoraproject.org/rpms/anaconda-user-help/blob/master/f/anaconda-user-help.spec :) It should be fixed now